### PR TITLE
Added Quill editor for event description

### PR DESCRIPTION
**Pull request by mjimenezmartin**
- Adds `Quill` editor for event description.
- Removes `ControlValueAccesor` from `DescriptionEditionComponent`

### CODE REVIEW

File: src/app/apps/TopicFriends/meetings/meeting-details-page/description-editor/description-editor.component.sass [hunk 1]
```diff
@@ -0,0 +1 @@
+
```
Comment (karol-depka) [inline src/app/apps/TopicFriends/meetings/meeting-details-page/description-editor/description-editor.component.sass]: Plz don't commit stuff like that ;)

File: src/app/apps/TopicFriends/topic-friends.module.ts [hunk 2]
```diff
@@ -1,12 +1,13 @@
 import { NgModule } from '@angular/core';
 import { CommonModule } from '@angular/common';
 import {TopicFriendsRoutingModule} from "./TopicFriends.routing.module";
+import { QuillModule } from 'ngx-quill';
```
Comment (karol-depka) [inline src/app/apps/TopicFriends/topic-friends.module.ts]: Plz don't commit stuff like that ;)

File: src/app/apps/TopicFriends/topic-friends.module.ts [hunk 3]
```diff
@@ -1,12 +1,13 @@
...
 
 @NgModule({
   declarations: [],
   imports: [
     CommonModule,
-    TopicFriendsRoutingModule,
+    TopicFriendsRoutingModule
```
Comment (karol-depka) [inline src/app/apps/TopicFriends/topic-friends.module.ts]: Plz don't commit stuff like that ;)

File: src/app/apps/TopicFriends/meetings/meeting-details-page/description-editor/description-editor.component.ts [hunk 4]
```diff
@@ -1,34 +1,17 @@
...
-      provide: NG_VALUE_ACCESSOR,
-      useExisting: forwardRef(() => DescriptionEditorComponent),
-      multi: true,
-    }
-  ],
 })
-export class DescriptionEditorComponent implements OnInit, ControlValueAccessor {
+export class DescriptionEditorComponent implements OnInit {
+
+  @Input() control: FormControl;
```
Comment (karol-depka) [inline src/app/apps/TopicFriends/meetings/meeting-details-page/description-editor/description-editor.component.ts]: :)

File: src/app/apps/TopicFriends/meetings/meeting-details-page/description-editor/description-editor.component.ts [hunk 5]
```diff
@@ -1,34 +1,17 @@
...
-import {ControlValueAccessor, NG_VALUE_ACCESSOR} from "@angular/forms";
+import { Component, forwardRef, Input, OnInit, ViewChild } from '@angular/core';
+import { ControlValueAccessor, FormControl, NG_VALUE_ACCESSOR } from "@angular/forms";
+import { QuillEditorComponent } from 'ngx-quill';
 
 @Component({
   selector: 'app-description-editor',
   templateUrl: './description-editor.component.html',
   styleUrls: ['./description-editor.component.sass'],
-  providers: [
```
Comment (karol-depka) [inline src/app/apps/TopicFriends/meetings/meeting-details-page/description-editor/description-editor.component.ts]: Thanks for Your take on the `ControlValueAcessor` vs `@Input`

File: src/global.scss [hunk 6]
```diff
@@ -9,6 +9,14 @@
 @import '~@ionic/angular/css/text-alignment.css';
 @import '~@ionic/angular/css/text-transformation.css';
 @import '~@ionic/angular/css/flex-utils.css';
+@import '~quill/dist/quill.core.css';
+@import '~quill/dist/quill.bubble.css';
+@import '~quill/dist/quill.snow.css';
+
+[contenteditable] {
+  -webkit-user-select: text;
```
Comment (karol-depka) [inline src/global.scss]: What is the purpose of this?
Is it for drag-and-drop of images?
Please add comment if possible :)

Comment (mjimenezmartin) [reply src/global.scss]: Required for QuillEditor. Need to discover why.

**Comment by mjimenezmartin** [comment]
I think we should keep the `@input` name as `formControl` instead of `control` to handle it in the same way as Angular does. WDYT @karol-depka ?

**Pull request merged**